docs/community-run-instances.md

```diff
@@ -440,6 +456,8 @@ Next to its native webclient interface, Jitsi Meet is also available as
 | meet.mayfirst.org |  ✅  |    |  [run SSL check!](https://www.ssllabs.com/ssltest/analyze.html?d=meet.mayfirst.org&latest) | Hurricane Electric LLC   |
 | team.video |  ✅  | ❌   |  [run SSL check!](https://www.ssllabs.com/ssltest/analyze.html?d=team.video&latest) |     AMAZON-AES  |
 | jitsi.member.fsf.org/ | ✅ | | [run SSL check!](https://www.ssllabs.com/ssltest/analyze.html?d=jitsi.member.fsf.org&latest) | Free Software Foundation |
+| swrtc.talky.io |   |  ❌ | [run SSL check!](https://www.ssllabs.com/ssltest/analyze.html?d=swrtc.talky.io&latest) | AMAZON-02 |
+
```

Comment:
This is not a Jitsi Meet instance.

Reply:
blimey, you're right, sorry about that - have removed that one ;)